CMakeLists.txt

```diff
@@ -58,12 +61,12 @@ option(LLAMA_SANITIZE_ADDRESS           "llama: enable address sanitizer"
 option(LLAMA_SANITIZE_UNDEFINED         "llama: enable undefined sanitizer"                     OFF)
 
 # instruction set specific
-option(LLAMA_AVX                        "llama: enable AVX"                                     ON)
-option(LLAMA_AVX2                       "llama: enable AVX2"                                    ON)
+option(LLAMA_AVX                        "llama: enable AVX"                                     OFF)
```

Comment:
I don't think we should be changing default values in a compatibility PR.

Reply:
Agreed. I need to separate out SSE3 as an option as well. I currently am testing this on a P4 without AVX, so it would be beneficial to test these.

Reply:
Addressed. SSE3 will come in later PR.